mtools/mplotqueries/mplotqueries.py

```diff
@@ -19,6 +19,7 @@
     from matplotlib.dates import AutoDateFormatter, date2num, AutoDateLocator
     from matplotlib import __version__ as mpl_version
     import mtools.mplotqueries.plottypes as plottypes
+    import re
```

Comment:
Review PEP style guide to sort imports https://www.python.org/dev/peps/pep-0008/#imports ;)
```suggestion
    import re
```